iot-ose(2).json

```diff
@@ -0,0 +1 @@
55799","2016-10-01 17:37:22"],["Vibration","1.5811316203379184","2016-10-01 17:37:22"],["Temperature","71.39146913196153","2016-10-01 17:37:27"],["Vibration","3.2032876747223593","2016-10-01 17:37:27"],["Temperature","72.76321428554955","2016-10-01 17:37:32"],["Vibration","4.88623593913888","2016-10-01 17:37:32"],["Temperature","73.22420837463602","2016-10-01 17:37:37"],["Vibration","6.7166067897074","2016-10-01 17:37:37"],["Temperature","74.89503733924926","2016-10-01 17:37:42"],["Vibration","8.117231768468947","2016-10-01 17:37:42"],["Temperature","50.0","2016-10-01 17:37:47"],["Vibration","8.839051119437478","2016-10-01 17:37:47"],["Temperature","50.96438634985279","2016-10-01 17:37:52"],["Vibration","9.252672113072043","2016-10-01 17:37:52"],["Temperature","52.1874857670886","2016-10-01 17:37:57"],["Vibration","10.697077023054993","2016-10-01 17:37:57"],["Temperature","52.3806829809301","2016-10-01 17:38:02"],["Vibration","11.04145742661182","2016-10-01 17:38:02"],["Temperature","54.07004541401591","2016-10-01 17:38:07"],["Vibration","11.908548538905364","2016-10-01 17:38:07"],["Temperature","55.10173603302251","2016-10-01 17:38:12"],["Vibration","12.889991207245895","2016-10-01 17:38:12"],["Temperature","56.51306466757221","2016-10-01 17:38:17"],["Vibration","14.69970727737479","2016-10-01 17:38:17"],["Temperature","56.66208014096139","2016-10-01 17:38:22"],["Vibration","15.644840275520009","2016-10-01 17:38:22"],["Temperature","58.50577660362472","2016-10-01 17:38:27"],["Vibration","16.612741139558544","2016-10-01 17:38:27"],["Temperature","59.40196637314997","2016-10-01 17:38:32"],["Vibration","17.054181175688157","2016-10-01 17:38:32"],["Temperature","59.90650206532234","2016-10-01 17:38:37"],["Vibration","17.705799721127956","2016-10-01 17:38:37"],["Temperature","61.50623499837298","2016-10-01 17:38:42"],["Vibration","0.0","2016-10-01 17:38:42"],["Temperature","62.80717623303909","2016-10-01 17:38:47"],["Vibration","0.29097079333317555","2016-10-01 17:38:47"],["Temperature","64.34283672189633","2016-10-01 17:38:52"],["Vibration","1.7429542120099748","2016-10-01 17:38:52"],["Temperature","66.24462713447625","2016-10-01 17:38:57"],["Vibration","1.8557530944468408","2016-10-01 17:38:57"],["Temperature","66.80794296792833","2016-10-01 17:39:02"],["Vibration","3.2055096270271974","2016-10-01 17:39:02"],["Temperature","68.78763215976333","2016-10-01 17:39:07"],["Vibration","3.6388992551578387","2016-10-01 17:39:07"],["Temperature","70.68267522446935","2016-10-01 17:39:12"],["Vibration","5.036571517017691","2016-10-01 17:39:12"],["Temperature","72.67343404915837","2016-10-01 17:39:17"],["Vibration","5.558358923970779","2016-10-01 17:39:17"],["Temperature","74.56998717903441","2016-10-01 17:39:22"],["Vibration","7.259834223242555","2016-10-01 17:39:22"],["Temperature","50.0","2016-10-01 17:39:27"],["Vibration","8.865494110818236","2016-10-01 17:39:27"],["Temperature","51.77669183235463","2016-10-01 17:39:32"],["Vibration","10.493133069852147","2
```

Comment:
I updated/reduced the content in my latest [PR](https://github.com/sabre1041/iot-ose/pull/10)